modules/function/src/test/java/com/opengamma/strata/function/calculation/swap/SwapPricingTest.java

```diff
@@ -146,11 +148,16 @@ public void presentValueVanillaFixedVsLibor1mSwap() {
     List<Column> columns = ImmutableList.of(Column.of(Measure.PRESENT_VALUE));
     ReportingRules reportingRules = ReportingRules.fixedCurrency(USD);
     CalculationRules rules = CalculationRules.of(pricingRules, marketDataRules, reportingRules);
-    CalculationRunner runner = CalculationRunnerFactory.ofSingleThreaded()
-        .createWithMarketDataBuilder(trades, columns, rules, marketDataFactory(), MarketDataConfig.empty());
 
     // calculate results using the runner
-    Results results = runner.calculateSingleScenario(suppliedData);
+    Results results = null;
+    try (CalculationTaskRunner runner = CalculationTaskRunner.of(MoreExecutors.newDirectExecutorService())) {
```

Comment:
No need to wrap it in a try-with-resources block if we're using a direct executor.